packages/@aws-cdk/aws-dynamodb/test/dynamodb.test.ts

```diff
@@ -345,6 +347,297 @@ test('when specifying every property', () => {
   );
 });
 
+test('when specifying sse with customer managed CMK', () => {
+  const stack = new Stack();
+  const table = new Table(stack, CONSTRUCT_NAME, {
+    tableName: TABLE_NAME,
+    encryption: TableEncryption.CUSTOMER_MANAGED,
```

Comment:
Would be nice to also have a test that you get this behavior automatically if you _only_ specify the `encryptionKey` property and nothing else.